src/pybind/mgr/dashboard/frontend/cypress/e2e/orchestrator/workflow/nfs/nfs-export.po.ts

```diff
@@ -3,21 +3,20 @@ import { PageHelper } from '../../../page-helper.po';
 /* tslint:enable*/
 
 const pages = {
-  index: { url: '#/nfs', id: 'cd-nfs-list' },
-  create: { url: '#/nfs/create', id: 'cd-nfs-form' }
+  cephfs_index: { url: '#cephfs/nfs', id: 'cd-nfs-list' },
+  cephfs_create: { url: '#cephfs/nfs/create', id: 'cd-nfs-form' },
+  rgw_index: { url: '#rgw/nfs', id: 'cd-nfs-list' },
+  rgw_create: { url: '#rgw/nfs/create', id: 'cd-nfs-form' }
 };
 
 export class NFSPageHelper extends PageHelper {
   pages = pages;
 
-  @PageHelper.restrictTo(pages.create.url)
+  @PageHelper.restrictTo(pages.cephfs_create.url)
```

Comment:
this view could be from either `rgw` or `cephfs`. So we can't really restrict it to a particular url. I think we only have e2es written for rgw exports so right now atleast the test should start from the `/rgw/nfs` form. We can remove this `restrictTo` from here to simplify it.

Reply:
I guess I missed this one to change

Reply:
Removed now!